tests/dummy/app/pods/integration-list/route.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember'
+
+export default Ember.Route.extend({
+
+  _fetch () {
+    return this.get('store').query('list-item', {pageSize: 10, start: 0})
```

Comment:
Actually we should demo both a small number 3-5 and a large number (full pageSize of 100-500)